src/main/java/io/cdap/e2e/pages/locators/CdfSchemaLocators.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.e2e.pages.locators;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+import java.util.List;
+
+/**
+ * Represents CdfSchemaLocators
+ */
+public class CdfSchemaLocators {
+  @FindBy(how = How.XPATH,
+    using = "//div[@data-cy='Output Schema']//div[@data-cy='schema-fields-list']//*[@placeholder='Field name']")
+  public static List<WebElement> outputSchemaColumnNames;
+
+  @FindBy(how = How.XPATH, using = "//div[@data-cy='Output Schema']//div[@data-cy='schema-fields-list']//select")
+  public static List<WebElement> outputSchemaDataTypes;
+
+  @FindBy(how = How.XPATH,
+    using = "//div[@data-cy='Input Schema']//div[@data-cy='schema-fields-list']//*[@placeholder='Field name']")
+  public static List<WebElement> inputSchemaColumnNames;
+
+  @FindBy(how = How.XPATH, using = "//div[@data-cy='Input Schema']//div[@data-cy='schema-fields-list']//select")
+  public static List<WebElement> inputSchemaDataTypes;
+
+  @FindBy(how = How.XPATH, using = "(//h2[text()='Input Records']/parent::div/div/div/div/div)[1]//div[text()!='']")
+  public static List<WebElement> previewInputRecordColumnNames;
```

Comment:
Can we move all the preview related locators to `CdfStudioLocators`?

Reply:
Moved to `CdfStudioLocators`